modulus/models/graphcast/graph_cast_net.py

```diff
@@ -132,8 +135,13 @@ class GraphCastNet(Module):
 
     Parameters
     ----------
-    multimesh_level: int, optional
-        Level of the multi-mesh, by default 6
+    mesh_level: int, optional
+        Level of the latent mesh, by default 6
+    multimesh: bool, optional
+        If the latent mesh is a multimesh, by default True
+        If True, the latent mesh includes the nodes corresponding
+        to the specified `mesh_level`and incorporates the edges from
+        all mesh levels ranging from level 0 up to and including `mesh_level`.
```

Comment:
I know this was asked above but it appears to me that renaming `multimesh_level` -> `mesh_level` would indeed break existing `GraphCastNet` checkpoints, no?

Reply:
Thanks for your comment. I added backward compatibility, and a deprecation warning.